tests/ui/conftest.py

```diff
@@ -77,14 +133,135 @@ def create_superuser(transactional_db, my_base_url, tmpdir, variables):
+    from olympia.addons.forms import icons
+    from olympia.addons.models import Addon, AddonCategory, Category, Preview, AddonUser
+    from olympia.addons.utils import generate_addon_guid
+    from olympia.constants.categories import CATEGORIES
+    from olympia.constants.applications import APPS, FIREFOX
+    from olympia.reviews.models import Review
+    from olympia.users.models import UserProfile
+    from olympia.landfill.collection import generate_collection
+    from olympia.landfill.user import generate_addon_user_and_category, generate_user
+    from olympia.landfill.categories import generate_categories
+    from olympia.constants.base import (
+        ADDON_EXTENSION, ADDON_PERSONA, STATUS_PUBLIC)
+
+    default_icons = [x[0] for x in icons() if x[0].startswith('icon/')]
+    addon = addon_factory(
+        status=STATUS_PUBLIC,
+        type=ADDON_EXTENSION,
+        average_daily_users=4242,
+        users=[UserProfile.objects.get(username='uitest')],
+        average_rating=4.21,
+        description=u'My Addon description',
+        file_kw={
+            'hash': 'fakehash',
+            'platform': amo.PLATFORM_ALL.id,
+            'size': 42,
+        },
+        guid=generate_addon_guid(),
+        homepage=u'https://www.example.org/',
+        icon_type=random.choice(default_icons),
+        name=u'Ui-Test',
+        public_stats=True,
+        slug='ui-test',
+        summary=u'My Addon summary',
+        support_email=u'support@example.org',
+        support_url=u'https://support.example.org/support/ui-test-addon/',
+        tags=['some_tag', 'another_tag', 'ui-testing', 'selenium', 'python'],
+        total_reviews=777,
+        weekly_downloads=2147483647,
+        developer_comments='This is a testing addon, used within pytest.',
+        is_experimental=True,
+    )
+    first_preview = Preview.objects.create(addon=addon, position=1)
+    Review.objects.create(addon=addon, rating=5, user=user_factory())
+    Review.objects.create(addon=addon, rating=3, user=user_factory())
+    Review.objects.create(addon=addon, rating=2, user=user_factory())
+    Review.objects.create(addon=addon, rating=1, user=user_factory())
+    addon.reload()
+    AddonUser.objects.create(user=user_factory(username='ui-tester2'),
+                             addon=addon, listed=True)
+    version_factory(addon=addon, file_kw={'status': amo.STATUS_BETA},
+                    version='1.1beta')
+    addon.save()
+    generate_collection(addon, app=FIREFOX,
+                        author=UserProfile.objects.get(username='uitest'),
+                        )
+
+    print('Created custom addon for testing successfully')
+
+
+@pytest.fixture
+def force_user_login():
+    """
+        Fixture for providing the user object to seleniumlogin for logging into
+        the live_server specifically. Does not run on 'dev' or 'stage'
+        environments.
+    """
```

Comment:
First line of the docstring should be a summary. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings